modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -574,6 +569,19 @@ private JSONArray _getItemsActionsJSONArray(
 			});
 	}
 
+	private String _getLabelFromProperties(
+		Map<String, Object> fdsFieldProperties, String defaultLabel,
+		String fallbackLabel) {
```

Comment:
nit: For clarity, I'd call these `defaultProperty` and `fallbackProperty`

Reply:
:+1: I will also change the order of the parameters to keep them alphabetical ordered